api/app/src/main/resources/db/migration/V20240820__initial_run_info.sql

```diff
@@ -0,0 +1,16 @@
+-- Create run info table
+CREATE TABLE IF NOT EXISTS "run_info"
+(
+    "task_id"             TEXT PRIMARY KEY NOT NULL,
+    "status"              TEXT,
+    "commit_hash"         TEXT,
+    "branch"              TEXT,
+    "logs"                TEXT,
```

Comment:
should this be json as well? since its stored that in in the entity?

Reply:
yep, good spot, thanks